start.py

```diff
@@ -345,27 +345,31 @@ def cm_server_not_dead(primary_node):
                                               cluster=cluster, quiet=not args.verbose)
 
 
-def _configure_kdc(cluster, kerberos_principals, quiet):
+def _configure_kdc(cluster, kerberos_principals, kerberos_ticket_lifetime, quiet):
     kdc_node = cluster.kdc_node
 
     logger.info('Updating KDC configurations ...')
     realm = cluster.network.upper()
 
     logger.debug('Updating krb5.conf ...')
     krb5_conf = cluster.kdc_node.get_file(KDC_KRB5_CONF_FILENAME)
-    kdc_node.put_file(KDC_KRB5_CONF_FILENAME,
-                      re.sub(r'EXAMPLE.COM', realm,
-                             re.sub(r'example.com', cluster.network,
-                                    re.sub(r'kerberos.example.com',
-                                           kdc_node.fqdn,
-                                           krb5_conf))))
+    ticket_lifetime_repl = kerberos_ticket_lifetime if kerberos_ticket_lifetime else '\g<1>'
```

Comment:
What is `_repl`? Can we name this variable in a human-readable way (here and below)?

Reply:
Done.